# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../owb/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- trunk/BAL/CMakeConf.txt.vanilla	2007-10-08 16:34:25.000000000 +0200
+++ trunk/BAL/CMakeConf.txt	2007-10-08 16:34:47.000000000 +0200
@@ -340,7 +340,7 @@
 
 IF (WEBKIT_USE_FONTS_FREETYPE OR WEBKIT_USE_FONTS_SDL_TTF)
 	SET (OWB_DEFAULT_FONT "Vera.ttf" CACHE STRING "Font selection for browser font-rendering")
-	EXEC_PROGRAM (find ARGS /usr/share/fonts/ -name ${OWB_DEFAULT_FONT} OUTPUT_VARIABLE OWB_DEFAULT_FONT)
+	EXEC_PROGRAM (find ARGS /usr/X11R7/lib/X11/fonts/TTF/ -name ${OWB_DEFAULT_FONT} OUTPUT_VARIABLE OWB_DEFAULT_FONT)
 	IF (EXISTS ${OWB_DEFAULT_FONT})
 		EXEC_PROGRAM (ln ARGS -sf ${OWB_DEFAULT_FONT} /usr/share/fonts/owb.ttf OUTPUT_VARIABLE OWB_DEFAULT_FONT)
 	ELSE (EXISTS ${OWB_DEFAULT_FONT})
